k8s/frontend-deployment.yml

```
@@ -0,0 +1,27 @@
+      labels: 
+        app: frontend
+    spec:
+      containers:
+      - name: chatapp-frontend
+        image: swarnendukar123/chatapp-frontend:latest
+        ports:
+        - containerPort: 80
+        env:
+        - name: NODE_ENV
+          value: production
+       
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)